staging_docs/user/guides/sync.md

```diff
@@ -0,0 +1,198 @@
+      "client_cert": null,
+      "tls_validation": true,
+      "proxy_url": null,
+      "pulp_labels": {},
+      "pulp_last_updated": "2021-03-09T04:14:02.646845Z",
+      "download_concurrency": 10,
+      "policy": "on_demand",
+      "total_timeout": null,
+      "connect_timeout": null,
+      "sock_connect_timeout": null,
+      "sock_read_timeout": null,
+      "headers": null,
+      "rate_limit": null,
+      "includes": [
+        "shelf-reader"
+      ],
+      "excludes": [],
+      "prereleases": true,
+    }
+    ```
+
+Reference: [Python Remote Usage](https://pulpproject.org/pulp_python/restapi/#tag/Remotes:-Python)
+
+## A More Complex Remote
+
+If only the name of a project is specified, every distribution of every version of that project
+will be synced. You can use the version_specifier field to ensure only distributions you care
+about will be synced:
+
+```bash
+pulp python remote create \
+    --name 'complex-remote' \
+    --url 'https://pypi.org/' \
+    --includes '[
+        "django~=2.0,!=2.0.1",
+        "pip-tools>=1.12,<=2.0",
+        "scipy",
+        "shelf-reader"
+    ]'
+```
+
+You can also use version specifiers to "exclude" certain versions of a project, like so:
+
+```bash
+pulp python remote create \
+    --name 'complex-remote' \
+    --url 'https://pypi.org/' \
+    --includes '[
+        "django",
+        "scipy"
+    ]' \
+    --excludes '[
+        "django~=1.0",
+        "scipy"
+    ]'
+```
+
+You can also filter packages by their type, platform and amount synced through the "package_types",
+"exclude_platforms", and "keep_latest_packages" fields respectively, like so:
+
+```bash
+pulp python remote create \
+    --name 'complex-filters' \
+    --url 'https://pypi.org/' \
+    --includes '["django"]' \
+    --package-types '["sdist", "bdist-wheel"]' # only sync sdist and bdist-wheel package types \
+    --exclude-platforms '["windows"]' # exclude any packages built for windows \
+    --keep-latest-packages 5 # keep the five latest versions
+```
+
+Reference: [Python Remote Usage](https://pulpproject.org/pulp_python/restapi/#tag/Remotes:-Python)
+
+
+
+### Creating a remote to sync all of PyPI
+
+A remote can be setup to sync all of PyPI by not specifying any included packages like so:
+
+```bash
+pulp python remote create \
+    --name 'PyPI-mirror' \
+    --url 'https://pypi.org/' \
+    --excludes '[
+        "django~=1.0",
+        "scipy"
+    ]'
+```
+
+By not setting the "includes" field Pulp will ask PyPI for all of its available packages to sync, minus the ones from
+the excludes field. Default Python remotes are created with syncing policy "on_demand" because the most common
+Python remotes involve syncing with PyPI which requires terabytes of disk space. This can be changed by
+modifying the "policy" field.
+
+Syncing all of PyPI can take a long time depending on your network and disk speeds. Check out
+[pull-through caching](publish.md#enable-pull-through-caching) 
```

Comment:
I think this won't work, Mike had a similar problem last week.

My theory is that some config makes the path be relative to the html structure (`guides/sync/index.html`, rather than the actual `guides` dir). So `../publish.md` would probably work ,but its weird. Or you can use the absolute links